### ISSUE TITLE
Deploy a SQL Client UI in the Dev UI

### ISSUE THREAD

**Issue by gastaldi** [quarkusio/quarkus#20238]
### Description

It would be really cool if the Dev Console provided an SQL Client UI to query the configured databases quickly

### Implementation ideas

[SQLPad](https://getsqlpad.com/) is a nice SQL Client Web UI that is provided as a Docker image and could be used in conjunction with TestContainers.

See an example using Docker Compose here: https://github.com/sqlpad/sqlpad/blob/master/docker-examples/postgres-docker-compose/docker-compose.yml

**Comment by Sanne** [comment]
yea I'd love that!

**Comment by gastaldi** [comment]
How does a processor know the configured DataSources (JDBC URL, username, password) in the application? Haven't found a build item containing this info

**Comment by Sanne** [comment]
such details are not known at build time, as they are configurable at runtime

**Comment by gastaldi** [comment]
> such details are not known at build time, as they are configurable at runtime

Right, so what's the proper way to have this information? Read from the MP Config? What about the datasources created from devservices?

**Comment by Sanne** [comment]
Hi @gastaldi , I'm not entirely sure as I've not contributed much to the Dev UI but if it works like code in Runtime you should be able to inject the datasource directly (already configured and started).

**Comment by gastaldi** [comment]
Already available in http://localhost:8080/q/dev-ui/quarkus-agroal/database-view
